index.js

```diff
@@ -0,0 +1,50 @@
+
+// . reduce ()
+// obj
+// we have ; 
+// arrays, properties, objects(string, numbers)
```

Comment:
Always clean up comments before committing your final version. They're fine in development but you don't want them merged into production code.